transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventLoop.java

```diff
@@ -196,6 +196,10 @@ private void processReady(int ready) {
             } else if (filter == Native.EVFILT_READ) {
                 // Check READ before EOF to ensure all data is read before shutting down the input.
                 unsafe.readReady(eventList.data(i));
+            } else if (filter == Native.EVFILT_SOCK) {
+                if ((eventList.fflags(i) & Native.NOTE_RDHUP) != 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1066)